crates/bevy_tasks/src/iter/mod.rs

```diff
@@ -16,6 +16,7 @@ where
     B: Iterator<Item = Self::Item> + Send,
     Self: Sized + Send,
 {
+    /// The type of item that is being iterated over.
     type Item;
```

Comment:
Hmm, I'm surprised this item is needed at all, since it is definitionally the same as `B::Item`

Reply:
I can replace `Self::Item` with `B::Item` but technically that would be a breaking change, which I'm explicitly trying to avoid in a docs PR.

Reply:
Ehhh, what the hell. Done.